ArchiSteamFarm.OfficialPlugins.Monitoring/MonitoringConfig.cs

```diff
@@ -0,0 +1,37 @@
+// ----------------------------------------------------------------------------------------------
+//     _                _      _  ____   _                           _____
+//    / \    _ __  ___ | |__  (_)/ ___| | |_  ___   __ _  _ __ ___  |  ___|__ _  _ __  _ __ ___
+//   / _ \  | '__|/ __|| '_ \ | |\___ \ | __|/ _ \ / _` || '_ ` _ \ | |_  / _` || '__|| '_ ` _ \
+//  / ___ \ | |  | (__ | | | || | ___) || |_|  __/| (_| || | | | | ||  _|| (_| || |   | | | | | |
+// /_/   \_\|_|   \___||_| |_||_||____/  \__|\___| \__,_||_| |_| |_||_|   \__,_||_|   |_| |_| |_|
+// ----------------------------------------------------------------------------------------------
+// |
+// Copyright 2015-2024 Łukasz "JustArchi" Domeradzki
+// Contact: JustArchi@JustArchi.net
+// |
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// |
+// http://www.apache.org/licenses/LICENSE-2.0
+// |
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Text.Json.Serialization;
+
+namespace ArchiSteamFarm.OfficialPlugins.Monitoring;
+
+internal sealed class MonitoringConfig {
+	internal const bool DefaultEnabled = false;
+
+	[JsonInclude]
+	[JsonRequired]
+	public bool Enabled { get; private set; } = DefaultEnabled;
+
+	[JsonConstructor]
+	internal MonitoringConfig() { }
+}
```

Comment:
Do we really need a config considering this plugin is not bundled by default? I think we can safely skip those settings and include them only where needed. The only reason I have it for STD is that it's bundled by default, and I don't want users to delete it every release when not interested - and I don't see any other value in this plugin being disabled to justify.

Reply:
We could potentially add `Uri? DashboardUri`, implement `IWebInterface` and add a `www` folder with just `index.html` that redirects to that uri.

Reply:
I'm talking about `Enabled` here, you can just assume `Enabled = true` since I don't see a point why somebody would load the plugin without intending to use it.